pcre-light.cabal

```diff
@@ -19,8 +19,10 @@ author:          Don Stewart
 maintainer:      Daniel Díaz <dhelta.diaz@gmail.com>
 cabal-version: >= 1.8.0
 build-type:      Simple
-tested-with:     GHC == 7.8.3
-extra-source-files: README.md
+tested-with:     GHC >= 7.10, GHC >= 8.0, GHC >= 8.4, GHC >= 8.6, GHC >= 8.8
```

Comment:
Shouldn't these be equalities? Or maybe `GHC >= 7.10 && <= 8.8`? Or maybe we don't need this at all? Anyway, having so many "greater or equal to" looks wrong to me.

Reply:
This field isn't used by anything anyways; mostly for descriptive purposes. I switched it to `==` anyways 👍